pom.xml

```diff
@@ -83,7 +83,7 @@
     <aws.sdk.version>1.11.133</aws.sdk.version>
     <bigquery.connector.hadoop2.version>0.10.2-hadoop2</bigquery.connector.hadoop2.version>
     <bouncycastle.version>1.56</bouncycastle.version>
-    <cdap.version>6.10.0-SNAPSHOT</cdap.version>
+    <cdap.version>6.11.0-SNAPSHOT</cdap.version>
```

Comment:
why this change?

Reply:
Please revert if not required.

Reply:
Actually we can change it to 6.10.0 instead since it is released now.

Reply:
changed to 6.10

Reply:
done